swarm/src/behaviour.rs

```diff
@@ -320,6 +320,10 @@ pub enum ToSwarm<TOutEvent, TInEvent> {
         /// Whether to close a specific or all connections to the given peer.
         connection: CloseConnection,
     },
+
+    /// TODO: more documentation needed
+    /// Reports external address of a remote peer to other `NetworkBehaviours`
+    NewExternalAddrOfPeer { peer_id: PeerId, addr: Multiaddr },
```

Comment:
Adding this is also a breaking change for exhaustive matches of this enum so we should also bump the minor version of this crate and add a changelog entry.

Reply:
:heavy_check_mark: